packages/suite/src/components/firmware/FirmwareInitial.tsx

```diff
@@ -114,6 +119,11 @@ export const FirmwareInitial = ({
         return <ConnectDevicePromptManager device={device} />;
     }
 
+    // Bitcoin-only firmware is only available on T2 from v2.0.8 - older devices must first upgrade to 2.1.1 which does not have a Bitcoin-only variant
+    const isBitcoinOnlyAvailable =
+        !!device.firmwareRelease &&
+        !versionUtils.isEqual(device.firmwareRelease?.release.version, [2, 1, 1]);
```

Comment:
2.1.1 is not the only one without bitcoin only. Anything older than 2.1.5 or 1.8.3 miss it.
 We could just look if `url_bitcoinonly` is available so it's more general and easier to read?
```suggestion
        !!device.firmwareRelease?.release.url_bitcoinonly;
```

Reply:
aha, now I see why it's like that - we need to look if the available update version has url_bitcoinonly and not the current one

Reply:
but that is exactly what you could find in `device.firmwareRelease.release` - info about new the available update version (i.e. info about 2.1.1 if you have 2.0.8), so I believe my suggestion is still valid

Reply:
Good thinking! 83cd8ed7b2b139db86982c63ef92e61392518cca